server/routerlicious/packages/services-core/src/throttler.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INackContent, NackErrorType } from "@fluidframework/protocol-definitions";
+
+export enum ThrottlerRequestType {
+    OpenSocketConn = "OpenSocketConn",
+    SubmitOp = "SubmitOp",
+    AlfredHttps = "AlfredHttps",
+    HistorianHttps = "HistorianHttps",
```

Comment:
Or make it more specific like, HistorianGetSummary

Reply:
Wondering what the pros/cons of going that granular with the types would be. I could see it being beneficial if we want to throttle particularly taxing REST endpoints to a greater extent than other endpoints within the same area. However, the downside would be that the throttling configuration would have to scale alongside the number of endpoints, so if a Historian endpoint is added, to go from say 2 to 3 endpoints, then the number of allowed rest calls to Historian per timeframe would increase by 50% (my math might be wrong but you get the idea)

Reply:
That's true. But there could be scenarios where we would want to have different throttling limits for different endpoints within the same service. 
If we make it granular, it will be easier to support such scenarios later.

Reply:
I think for now, I would stick to more generic. Historian calls are not straight forward. For example, A get summary is multiple calls. First you get the version, then you get the tree, then the blobs. For now, sticking to just one level for historian is better. We can later update it become more granular.